code128.py

```diff
@@ -0,0 +1,66 @@
+# taken from https://bitbucket.org/whitie/python-barcode
+
```

Comment:
this code is taken from another python project that generates CODE128-style barcodes.  it's just the list of characters that are acceptable to put on our barcodes.